index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  
+    <header class="app-header">
+        <nav>
+        <a href="#">My Logo</a>
+        <i class="fa fa-bars fa-2x bars "></i>
+        </nav>
+    </header>
+    <div class="content img">
+        <p class="copy">I'm José</br>
+        Glad to see </br>
+        you
+        </p>
```

Comment:
Since this is the main title of this section, it would be better to use an `h1` tag. This helps in improving your website to increase its visibility when people search for products or services (SEO optimization).
You can even break it into parts so that you use `h1` tag for `I'm Jose` and `h2` tag for `Glad to see you`

Reply:
Good job following the advice of the previous reviewer